seedbuilder/areas.ori

```diff
@@ -1793,16 +1793,24 @@ home: MoonGrottoAboveTeleporter
 	pickup: LeftGrottoTeleporterExp
 		casual-core WallJump DoubleJump
 		casual-core Climb DoubleJump ChargeJump
+		standard-abilities DoubleJump Dash Ability=3
 		expert-core Grenade Bash
 		expert-core Climb ChargeJump
+		expert-core DoubleJump Dash
 		expert-dboost ChargeJump Health=4
 		expert-dboost WallJump Glide Health=4
 		expert-dboost Climb Glide Health=4
 		expert-dboost DoubleJump Glide Health=4
 		expert-dboost WallJump Health=7
 		expert-dboost Climb Health=7
+		expert-dboost Climb DoubleJump Health=4
+		expert-dboost Climb Dash Ability=3 Health=4
+		expert-dboost WallJump Dash Ability=3 Health=4
```

Comment:
I think all three of these can be in `standard-dboost`:
* `Climb DoubleJump Health=4`
* `Climb Dash Ability=3 Health=4`
* `WallJump Dash Ability=3 Health=4`

In all three cases, getting up the spikes at the end is not difficult (especially with Wall Jump, which vaults you up practically for free), and both Double Jump and Air Dash allow you to return without taking damage.

Reply:
nb: the `Climb DoubleJump Health=4` path can be executed without a ramp; just jump across the gap and faceplant the spikes, then hold climb and up.